### PR TITLE
Update `checkstyle` workflow env to ubuntu-20.04

### PR DESCRIPTION
**Pull request by szubersk**
### Motivation and Context

Update `checkstyle` workflow environment to ubuntu-20.04 https://github.com/openzfs/zfs/issues/12711

### Description

- `checkstyle` workflow uses ubuntu-20.04 environment
- improved `mancheck.sh` readability

### How Has This Been Tested?

make checkstyle
make lint

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [x] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Library ABI change (libzfs, libzfs\_core, libnvpair, libuutil and libzfsbootenv)
- [ ] Documentation (a change to man pages or other documentation)

### Checklist:

- [x] My code follows the OpenZFS [code style requirements](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**contributing** document](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/openzfs/zfs/tree/master/tests) to cover my changes.
- [x] I have run the ZFS Test Suite with this change applied.
- [x] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

### CODE REVIEW

File: module/Makefile.in [hunk 1]
```diff
@@ -115,7 +115,8 @@ modules_uninstall: modules_uninstall-@ac_system@
 
 cppcheck-Linux:
 	@CPPCHECK@ -j@CPU_COUNT@ --std=c99 --quiet --force --error-exitcode=2 \
-		--inline-suppr --suppress=noValidConfiguration \
+		--inline-suppr --suppress=unmatchedSuppression \
+		--suppress=noValidConfiguration \
```
Comment (ahrens) [inline module/Makefile.in:119]: [nit] it would be easier to read changes to this in the future if we formatted it with one flag per line

**Review by ahrens — APPROVED** [review]

File: module/Makefile.in (hunk 1 above)
Comment (szubersk) [reply module/Makefile.in:120]: Corrected in https://github.com/openzfs/zfs/commit/b9826acd531351530e163e8da5768cb1b61fb8e7

**Comment by szubersk** [comment]
@ahrens @jwk404 Might I ask for `checkstyle` workflow run? It's required to have it done before merge.

**Comment by szubersk** [comment]
Thanks!
Since
> You’re not authorized to merge this pull request.
are we ready to merge this PR in?

**Review by jwk404 — APPROVED** [review]

**Pull request merged**